app/views/admin/orders/show.html.erb

```diff
@@ -37,9 +37,13 @@
   <p><b>Order Date:</b> <%= @order.date %><br />
   <b>Order Placed by:</b> <%= @order.user.full_name %><br />
```

Comment:
merge conflicts here

Reply:
remove this part :  <b>Order Status:</b> <%= @order.status.capitalize %></p>